mondey_backend/src/mondey_backend/routers/users.py

```diff
@@ -152,4 +156,59 @@ def update_milestone_answer(
             session.commit()
         return milestone_answer
 
+    # Endpoints for answers to user question
+    @router.get("/user-answers/", response_model=list[UserAnswerPublic])
+    def get_current_user_answers(
+        session: SessionDep, current_active_user: CurrentActiveUserDep
+    ):
+        answers = session.exec(
+            select(UserAnswer).where(col(UserAnswer.user_id) == current_active_user.id)
+        ).all()
+
+        user_answers = [
+            UserAnswerPublic(
+                answer=answer.answer,
+                non_standard=answer.non_standard,
+                question_id=answer.question_id,
+            )
+            for answer in answers
+        ]
+        return user_answers
+
+    @router.put("/user-answers/", response_model=list[UserAnswerPublic])
+    def update_current_user_answers(
+        session: SessionDep,
+        current_active_user: CurrentActiveUserDep,
+        new_answers: list[UserAnswerPublic],
+    ):
+        if session is None:
+            raise HTTPException(401, detail="no session")
```

Comment:
is this needed - i.e. can session be none at this point?

Reply:
hm, it did make things a bit easier, but since the usage of userID has reduced by now to the `submitData` function I retrieve it there now.

Reply:
what guarantees(!) us that it can not? removing this afais would result in a  code 500 error without useful message.